### ISSUE TITLE
Spyder Not Launching from Anaconda-Navigator

### ISSUE THREAD

**Issue by irahulcse** [ContinuumIO/anaconda-issues#10142]
### Actual Behavior
When before I used to click on the Spyder it launche automatically but now it is not happening so.
This image comes after it closes down. I am able to open the Jupyter notebook but only the Spyder is not opening properly.
![screenshot from 2018-10-03 18-17-51](https://user-images.githubusercontent.com/34218254/46411244-dd517800-c738-11e8-831a-259b474a0314.png)

##### Anaconda or Miniconda version:
Anaconda 3 
##### Operating System:
![operating system](https://user-images.githubusercontent.com/34218254/46411106-88156680-c738-11e8-9a68-166449d996ed.png)

##### `conda info`

![screenshot from 2018-10-03 18-20-12](https://user-images.githubusercontent.com/34218254/46411290-05d97200-c739-11e8-8bd6-98bc2980332d.png)

```

```

##### `conda list --show-channel-urls`

```

```

**Comment by ccordoba12** [comment]
Please start Spyder from a terminal and paste here any errors that appear there.

**Comment by azureAxolotl** [comment]
Hi, I am seeing the same issue attempting to launch Spyder from anaconda-navigator. The Spyder "splash" appears for a few seconds before disappearing. Nothing else appears on screen, although quitting anaconda-navigator shows a prompt to say that Spyder is running.
:~$ neofetch
            .-/+oossssoo+/-.               xxxx@xxxxxxxxxxxxxxxxx 
        `:+ssssssssssssssssss+:`           --------------------------- 
      -+ssssssssssssssssssyyssss+-         OS: Ubuntu 18.04.1 LTS x86_64 
    .ossssssssssssssssssdMMMNysssso.       Host: GA-78LMT-USB3 6.0 
   /ssssssssssshdmmNNmmyNMMMMhssssss/      Kernel: 4.15.0-36-generic 
  +ssssssssshmydMMMMMMMNddddyssssssss+     Uptime: 45 mins 
 /sssssssshNMMMyhhyyyyhmNMMMNhssssssss/    Packages: 2733 
.ssssssssdMMMNhsssssssssshNMMMdssssssss.   Shell: bash 4.4.19 
+sssshhhyNMMNyssssssssssssyNMMMysssssss+   Resolution: 1680x1050 
ossyNMMMNyMMhsssssssssssssshmmmhssssssso   DE: GNOME 3.28.3 
ossyNMMMNyMMhsssssssssssssshmmmhssssssso   WM: GNOME Shell 
+sssshhhyNMMNyssssssssssssyNMMMysssssss+   WM Theme: Adwaita 
.ssssssssdMMMNhsssssssssshNMMMdssssssss.   Theme: Adwaita [GTK2/3] 
 /sssssssshNMMMyhhyyyyhdNMMMNhssssssss/    Icons: Adwaita [GTK2/3] 
  +sssssssssdmydMMMMMMMMddddyssssssss+     Terminal: gnome-terminal 
   /ssssssssssshdmNNNNmyNMMMMhssssss/      CPU: AMD FX-4300 (4) @ 3.800GHz 
    .ossssssssssssssssssdMMMNysssso.       GPU: AMD Radeon 3000 
      -+sssssssssssssssssyyyssss+-         Memory: 2297MiB / 15518MiB 
        `:+ssssssssssssssssss+:` 
            .-/+oossssoo+/-.                                       
:~$ conda info

     active environment : None
       user config file : /home/olaf/.condarc
 populated config files : /home/olaf/.condarc
          conda version : 4.5.11
    conda-build version : 3.10.5
         python version : 3.6.5.final.0
       base environment : /home/olaf/anaconda3  (writable)
           channel URLs : https://repo.anaconda.com/pkgs/main/linux-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/free/linux-64
                          https://repo.anaconda.com/pkgs/free/noarch
                          https://repo.anaconda.com/pkgs/r/linux-64
                          https://repo.anaconda.com/pkgs/r/noarch
                          https://repo.anaconda.com/pkgs/pro/linux-64
                          https://repo.anaconda.com/pkgs/pro/noarch
          package cache : /home/olaf/anaconda3/pkgs
                          /home/olaf/.conda/pkgs
       envs directories : /home/olaf/anaconda3/envs
                          /home/olaf/.conda/envs
               platform : linux-64
             user-agent : conda/4.5.11 requests/2.18.4 CPython/3.6.5 Linux/4.15.0-36-generic ubuntu/18.04 glibc/2.27
                UID:GID : 1000:1000
             netrc file : None
           offline mode : False
:~$ spyder
Received signal 11 SEGV_MAPERR 000000000108
#0 0x7fb4fbd7a72f <unknown>
#1 0x7fb4fa7ac79d <unknown>
#2 0x7fb4fbd7ac3e <unknown>
#3 0x7fb5271fa890 <unknown>
#4 0x7fb4f1776954 <unknown>
#5 0x7fb4f1777be8 <unknown>
#6 0x7fb5028ec914 glXCreatePbuffer
#7 0x7fb4fa80e6e9 <unknown>
#8 0x7fb4fa7f9748 <unknown>
#9 0x7fb4fc8b67bd <unknown>
#10 0x7fb4faafd111 <unknown>
#11 0x7fb4faafe905 <unknown>
#12 0x7fb4fcd6cb77 <unknown>
#13 0x7fb4fe5a37d5 <unknown>
#14 0x7fb4fbde30e5 <unknown>
#15 0x7fb4fbdde480 <unknown>
#16 0x7fb5271ef6db start_thread
#17 0x7fb526f1188f clone
  r8: 0000000000000000  r9: 0000000000000001 r10: 00007fb526f5a430 r11: 0000000000000206
 r12: 0000558e7021bf40 r13: 0000558e7021bf40 r14: 0000000000000005 r15: 0000558e70155bb0
  di: 0000558e70114590  si: 0000000000000000  bp: 0000558e70114590  bx: 0000000000000000
  dx: 0000000000008041  ax: ffffffffffffff50  cx: 0000000000000000  sp: 00007fb477ffe480
  ip: 00007fb4f1776954 efl: 0000000000010202 cgf: 002b000000000033 erf: 0000000000000004
 trp: 000000000000000e msk: 0000000000000000 cr2: 0000000000000108
[end of stack trace]
Calling _exit(1). Core file will not be generated.

**Comment by ccordoba12** [comment]
It looks like a problem in Qt (our graphical toolkit), not Spyder.

@mingwandroid, have you seen this before?

**Comment by mingwandroid** [comment]
No. I suspect poor graphics hardware or drivers or a lack of drivers though.

**Comment by AbhiSawant** [comment]
Even I am facing the same issue. "core file will not be generated"

**Comment by AbhiSawant** [comment]
This helps:
conda upgrade qt
